package.json

```diff
@@ -68,6 +68,7 @@
     "rehype-mathjax": "^6.0.0",
     "rehype-pretty-code": "^0.13.0",
     "rehype-raw": "^7.0.0",
+    "rehype-rewrite": "^4.0.2",
```

Comment:
let's avoid introducing a new dependency for this, we have https://github.com/syntax-tree/unist-util-visit already

Reply:
Sure, could you give me a pointer for how I could rewrite this with `unist-util-visit`? Is there maybe some code already in the repo that does something similar. Sorry if my question seems trivial - I'm not very familiar with the whole unified ecosystem.

Reply:
I think I managed. Feel free to take a look :)